tests/post/features/dns_resolution.feature

```diff
@@ -0,0 +1,5 @@
+@ci @local @post
+Feature: logs should be accessible
```

Comment:
I'm not sure to understand the name of the feature because the scenario seems quite different